samples/kvsWebRTCClientMasterGstSample.c

```diff
@@ -326,7 +326,7 @@ INT32 main(INT32 argc, CHAR* argv[])
 
     pSampleConfiguration->videoSource = sendGstreamerAudioVideo;
     pSampleConfiguration->mediaType = SAMPLE_STREAMING_VIDEO_ONLY;
-    pSampleConfiguration->receiveAudioVideoSource = receiveGstreamerAudioVideo;
+    pSampleConfiguration->receiveAudioVideoSource = NULL;
```

Comment:
Can we move the null initialization into the `createSampleConfiguration` function

Reply:
If we do that, we would be changing it for other samples that use this as well and we don't want to do that is why we are doing this outside

Reply:
Don't they all set it after invoking create though?

Reply:
Yes, but they set it to different receivers